schema/CompoundSchema.mcf

```diff
@@ -578,3 +578,150 @@ description: "A reliability score is manually set for all genes and indicates th
 descriptionUrl: "https://www.proteinatlas.org/about/assays+annotation#if_reliability_score"
 rangeIncludes: dcs:HumanProteinOccurrenceReliabilityEnum
 domainIncludes: dcs:HumanProteinOccurrence
+
+Node: dcid:Antibody
+name: "Antibody"
+typeOf: schema:Class
+subClassOf: dcs:Protein
+description: "An antibody is a kind of protective protein which is produced by the immune system when substances from the outer source invade the living organisms. The process of producing the antibodies is called immune response."
+
+Node: dcid:Antigen
+name: "Antigen"
+typeOf: schema:Class
+subClassOf: dcs:ChemicalCompound
+description: "The invading substances from the outer source which causes the immune response. It refers to a molecule or molecule structure which may interact with the immune cell antigen receptor or antibody. "
+descriptionUrl:"https://en.wikipedia.org/wiki/Antigen"
+
+Node: dcid:recognizesAntigen
+typeOf: schema:Property
+name: "recognizesAntigen"
```

Comment:
I'm slightly wondering about the naming of this property, specifically "recognizes". Is the target the "recognizedAntigen" or "targetedAntigen"?